src/components/connect.js

```diff
@@ -194,14 +322,14 @@ export default function connect(mapPropsToRequestsToProps, options = {}) {
         if (mapping.hasOwnProperty('value')) {
           return onFulfillment(meta)(mapping.value)
         } else {
-          const request = buildRequest(mapping)
+          const request = defaults.buildRequest(mapping)
           meta.request = request
           this.setAtomicState(prop, startedAt, mapping, initPS(meta))
 
-          const fetched = window.fetch(request)
+          const fetched = defaults.fetch(request)
```

Comment:
Oh, you might be able to use `bind` instead of `call`. I'm guessing that'll be easier to implement. See http://stackoverflow.com/a/34888425/129492